### PR TITLE
Add a CLI helper

### PR DESCRIPTION
**Pull request by felixonmars**
This is useful for applying simple rules on demand.

### CODE REVIEW

**Review by lilydjwg — CHANGES_REQUESTED** [review]
Please wrap the main code in a function (preferably named "main"). 

Setting `HOME` env may be unwanted. It's better to use another env to tell openredir where to find the script.

**Comment by felixonmars** [comment]
> Please wrap the main code in a function (preferably named "main").
> 
> Setting `HOME` env may be unwanted. It's better to use another env to tell openredir where to find the script.

Both addressed. Sorry for the late response.

File: openredir-cli [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        metavar="COMMAND",
+        nargs="+",
+        help="Command to run")
+
+    options = parser.parse_args()
+
+    LUA_SRC = "function redirect(f, s)"
+
+    for rule in options.rule:
+        LUA_SRC += 's = s:gsub("{}", "{}")'.format(*rule.split(":"))
```
Comment (lilydjwg) [inline openredir-cli:28]: '"' needs to be escaped. Also the replacement should happen at the start (if only absolution path is supported).

File: libopenredir.c.in [hunk 2]
```diff
@@ -84,7 +84,11 @@ void lib_init() {
   L = luaL_newstate();
   luaL_openlibs(L);
   char config[PATH_MAX];
-  strcpy(config, getenv("HOME"));
+  if (getenv("OPENREDIR_CONFDIR"))
+    strcpy(config, getenv("OPENREDIR_CONFDIR"));
```
Comment (lilydjwg) [inline libopenredir.c.in]: Let's use `strncpy` instead.

**Review by lilydjwg — COMMENTED** [review]
There are some not very important issues, but better get them resolved.

File: openredir-cli (hunk 1 above)
Comment (felixonmars) [reply openredir-cli:28]: Actually I think I can do Lua pattern match here? So perhaps escaping is not really mandatory... I also want to support partial names if possible.

Comment (lilydjwg) [reply openredir-cli:28]: Ahhh yes. It's still needed for the user to learn Lua :-)

**Pull request merged**